### ISSUE TITLE
VSN normalization also for TMT data

### ISSUE THREAD

**Issue by fstein** [Nesvilab/FragPipe-Analyst-doc#21]
Dear FragPipe-Analyst developers team,
is there a particular reason why VSN normalization is allowed for DIA and LFQ only? I have pretty good experience to apply VSN normalization to TMT data, at least if all measured conditions were measured in the same TMT set. When normalizing different TMT sets with VSN, it's indeed problematic with a lot of missing values. 
Would it be possible to enable VSN normalization for a limited number of TMT sets? Or put a warning there? It would also be great to see quantile normalization as an option for the normalization type. 

Thanks a lot for the consideration.

Best wishes from Heidelberg,

Frank

**Comment by hsiaoyi0504** [comment]
@fstein Thanks for the feedback. Do you mean that you suggest apply VSN to samples within the sample TMT plex set? In terms of quantile normalization, are you thinking rather than median centering at 50%, using other percentage as the centering point?

**Comment by fstein** [comment]
@hsiaoyi0504 Yes, we apply VSN to samples within the same TMT plex. We also apply VSN to samples from different TMT plexes if the different plexes just refer to different replicates of conditions in the same plex. In terms of quantile normalization, I am thinking of centering at the 50 % quantile (median). Also a simple median normalization typically does the same job. It would be just great to have any normalization method as an option in FragPipe-Analyst for TMT data.

**Comment by hsiaoyi0504** [comment]
@fstein Thanks for the feedback. In FragPipe workflows, TMT-Integrator generates different normalization including median centering already, so I think most users don't need to do that in the FragPipe-Analyst, but should ok to add it as well.

**Comment by fstein** [comment]
Since you can turn normalization in TMT-Integrator off, it is a nice thing to see the effect of normalization on the analysis in FragPipe Analyst. 
If it would be christmas, I would also ask if you could include not only VSN normalization, but also a simple median and quantile normalization into the Normalization types in FragPipe Analyst.
Thanks for your consideration.

**Comment by hsiaoyi0504** [comment]
@fstein median centered normalizaion is implemented. We are still evaluating vsn or combat/limma approach for inter TMT-plex normalization.

**Comment by fstein** [comment]
This is great news. Thanks a lot. 
For inter TMT-plex normalization when having a lot of missing values, quantile normalization works better than vsn. However, for TMT both VSN (without many missing values) or quantile normalization works similarly well as median normalization since you don't really have this mean-variance dependency as for example in DIA or label free or even worth with microarray data.
